ui/src/dashboards/selectors/index.ts

```diff
@@ -41,6 +41,9 @@ export const sortDashboardByName = (dashboards: Dashboard[]): Dashboard[] =>
     a.name.toLowerCase() > b.name.toLowerCase() ? 1 : -1
   )
 
+export const isDashboardActive = (state: AppState): boolean =>
+  state.currentPage === 'dashboard'
```

Comment:
can you drop a comment in here, or next to the usage, about why we have to track this?

Reply:
Definitely